addon-test-support/@ember/test-helpers/dom/-is-form-control.js

```diff
@@ -0,0 +1,11 @@
+const FORM_CONTROL_TAGS = ['INPUT', 'BUTTON', 'SELECT', 'TEXTAREA'];
+
+export default function isFormControl(el) {
+  let { tagName, type } = el;
+
+  if (type === 'hidden') {
+    return false;
+  }
```

Comment:
I think we could improve the logic here. I don't think that disabled buttons or inputs should be focusable.
I found this nice table we can copy from, even if we're not soooo exhaustive: https://allyjs.io/data-tables/focusable.html